app/form_models/vacancy_algolia_search_form.rb

```diff
@@ -9,10 +9,10 @@ class VacancyAlgoliaSearchForm
               :total_filters
 
   def initialize(params = {})
-    @keyword = params[:keyword]
+    @keyword = params[:keyword].strip!
```

Comment:
How about just doing?

`params.values.each(&:strip!)`

Reply:
I can't think of any scenario where we wouldn't want to strip whitespace, so maybe we should do this at a controller level so params are stripped by default?